src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ProjectView.xaml.cs

```diff
@@ -248,10 +249,13 @@ private void SetComboboxCurrentVersion(string comboboxText, IEnumerable<NuGetVer
             for (int i = 0; i < _versions.Items.Count; i++)
             {
                 DisplayVersion currentItem = _versions.Items[i] as DisplayVersion;
-                if (currentItem != null && (comboboxText == _versions.Items[i].ToString() || _versions.Items[i].ToString() == matchVersion?.ToString()))
+                if (currentItem != null &&
+                    _versions.Items[i] != null && // null separator
+                    (comboboxText.Trim() == currentItem.Version.ToNormalizedString() || // trim extra spaces and compare versions without labels
+                     matchVersion?.ToString() == currentItem.Version.ToNormalizedString()))
```

Comment:
I see that `currentItem.Version.ToNormalizedString()` is invoked twice in this code snippet. Can we just invoke the method once and cache the result so that we can use the cached value for comparison?